caluma-demo/app/templates/documents/index.hbs

```diff
@@ -22,7 +22,7 @@
         </th>
       </tr>
     </thead>
-    {{#each this.model as |document|}}
+    {{#each (sort-by "node.createdAt:desc" this.model) as |document|}}
```

Comment:
Is there no sort option in the graphQL reqeust? I think it would be nicer to get it sorted from the API instead of sorting it in the frontend.

Reply:
There is a orderBy option in the GraphQL request. I now used this option for sorting from the API.